ui/.gitignore

```diff
@@ -0,0 +1,37 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
I think this already exists in the parent directory? maybe revert or merge if there is more stuff?

Reply:
Yup. done.